cores/esp32/Arduino.h

```diff
@@ -79,10 +79,9 @@
 #define degrees(rad) ((rad)*RAD_TO_DEG)
 #define sq(x) ((x)*(x))
 
-#define sei()
-#define cli()
```

Comment:
You could have define sei and cli instead for backwards compatibility with older code. Any reason you didn't @SuGlider?

Reply:
As far as I know, SEI and CLI are ASM instructions used in ATMEL chips. 
Arduino official API is `interrupts()` and `noInterrupts()`
I see no reason for defining those ATMEL specific functions.

``` cpp
//interrupts is defined as:

#define interrupts() sei()
//and sei is defined as for the UNO/328p:

# define sei()  __asm__ __volatile__ ("sei" ::)
```

Reply:
Looking ESP8266 `Arduino.h` I see that it also doesn't define `sei()` nor `cli()`.
My vote is for keeing it as defined in this PR.

Reply:
`interrupts()` and `noInterrupts()` are described in [Arduino Language Reference](https://www.arduino.cc/reference/en/language/functions/interrupts/nointerrupts/) so I vote for this too.